### ISSUE TITLE
Issue reporter various accessibility concerns

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#42570]
### Issue Type
Bug

#42351

### Description

I didn't actually test with a screen reader as I tested on Linux but I have the following concerns:

- "Please complete the form in English." appears to only be exposed to screen readers?
- The `#description-validation-error` looks like it should have `role="alert"` to alert screen readers

### VS Code Info

VS Code version: Code - Insiders 1.20.0-insider (8cac1acd97466afc75c663dc1626d8562a9a3f3d, 2018-01-30T17:28:33.081Z)
OS version: Linux x64 4.13.0-31-generic

Reproduces only with extensions

**Comment by RMacfarlane** [comment]
As discussed, "Please complete the form in English" has display: none, so I'm only adding the `role="alert"`
